src/control/cad.js

```diff
@@ -253,103 +261,265 @@ class CadControl extends Control {
       features.push(featureDict[dists[i]]);
     }
 
+    const drawFeature = this.editor.getDrawFeature();
+    if (drawFeature) {
+      // Include all but the last vertex (at mouse position) to prevent snapping on mouse cursor node
+      const currentDrawFeature = drawFeature.clone();
+      currentDrawFeature
+        .getGeometry()
+        .setCoordinates(
+          drawFeature.getGeometry().getCoordinates().slice(0, -1),
+        );
+      features = [currentDrawFeature, ...features];
+    }
+
+    const editFeature = this.editor.getEditFeature();
+    if (editFeature && this.editor.modifyStartCoordinate) {
```

Comment:
this.editor.modifyStartCoordinate this property seems useless. Isn't it the same value as in `evt.coordinate` ? I guess if editFeature exists , that means the node under the mouse coordinate is modifiying.

Reply:
Agreed